### DISCUSSION TITLE
Song select Ascending/Descending sort order

### DISCUSSION THREAD

**Discussion by Mux1c** [ppy/osu#26291] [Ideas]
I'm honestly surprised it's not in yet, but I believe it would be nice to choose ascending or descending order when sorting beatmaps, as for e.g. back in stable, when sorting maps by date added, the newest ones are at the bottom(or at least that's how I remember it to be), but in lazer, the newest maps are at the top, it's a small thing, but it takes adjusting to, and I think it would be cool for people to choose what they're comfortably with

**Comment by Rekunan** [comment]
Ik they try their best to not have any extra toggles/options, so maybe this could be set in place through a community poll of some sort instead.
And if there is somehow a new toggle/option for this, I disagree with using ascending/descending terminology, because the wording can be confusing (ascending as in increasing with how recent the map was added or how long ago it was).

**Reply by Mux1c** [reply]
Late reply as I don't use github often, but I see you've retracted you're statement on the extra feature comment, so I won't say anything about that. As for the ascending/descending terminology, I believe it fits best as it can be used for most sort options without need for clarification as opposed to other options like A to Z, Lowest to Highest, etc.

**Comment by WitherFlower** [comment]
I 100% agree with this proposal, and will add that this functionality already exists in the beatmap listing, using up and down arrows to represent the order, so there is no reason for it not to be added to song select as well.

![image](https://github.com/ppy/osu/assets/52672543/11b81fe2-3282-4b18-87ef-72ac47ecba1a)
![image](https://github.com/ppy/osu/assets/52672543/c6f9af3c-b040-4540-aa59-ec1721baac04)

**Reply by Rekunan** [reply]
Forgot about that, oops.

Since that's so easy to use, I no longer disagree from earlier, since someone can just click on it again.

**Comment by NoelleTGS** [comment]
Is this still being looked at for SSv2? I think the option would fit nicely as radio buttons under one of these buttons from the Figma that I don't think have been implemented
![image](https://github.com/user-attachments/assets/d91faa85-5345-45ac-92f3-1cf575080b6b)
